NEWS.md

```diff
@@ -6,10 +6,67 @@ Version X.Y of OpenDDS.
 - Add support for versioned namespaces
 - Rework type support implementation in order to expose less internal header
   and classes to user code
+- Add read/take_instance_w_condition
+- Time Based Filter QoS 
```

Comment:
Just wondering if it should be "Time Based Filter QOS support for Reliable Data Reader" or something like that, since Time Based Filter QOS was already in place, just not complete?